pyproject.toml

```diff
@@ -49,6 +49,7 @@ dependencies = [
     "pandas >= 2.1",
     "sdds >= 0.4",
     "h5py >= 2.9",
+    "tfs-pandas >= 4.0.0", # for madng (potentially an optional dependency)
```

Comment:
Writing on modified line: let's drop 3.9. Also should bump to a feature version in the `__init__.py`